v0/destinations/facebook_pixel/transform.js

```diff
@@ -146,12 +151,13 @@ const handleProductListViewed = (message, categoryToContent) => {
   if (products && products.length > 0 && Array.isArray(products)) {
     products.forEach(product => {
       if (isObject(product)) {
-        const productId = product.product_id;
+        const productId = product.product_id || product.sku || product.id || "";
```

Comment:
Have we checked if `empty string` is valid or not?

Reply:
Yes, empty string is not valid. We are removing it later on. and throwing error accordingly